data_processing/radiology/cli/load_dicom.py

```diff
@@ -0,0 +1,71 @@
+'''
+Created: February 2021
+@author: aukermaa@mskcc.org
+
+Given a scan (container) ID
+1. resolve the path to the dicom folder
+2. for all dicoms, rescale into HU and optionally window
```

Comment:
is HU rescale still being done in this file, or has it been refactored?

Reply:
Ahh, fixed this doc string